processing/src/main/java/io/druid/segment/data/CompressedLongsIndexedSupplier.java

```diff
@@ -218,6 +218,8 @@ public void remove()
 
   private class CompressedIndexedLongs implements IndexedLongs
   {
+    final CloseableIndexed<ResourceHolder<LongBuffer>> cachedLongBuffers = baseLongBuffers.getCachingIndexed(0);
```

Comment:
i found this a little confusing and had to look at the getCachingIndexed comments to understand what was happening. Is there a way to have another method for the special case when lookupCacheSize == 0?